#try-except block
# try:
#     print(x)
# except:
#     print("hi")


# try:
#     print(x)
# except NameError:
#     print("hi")
# except:
#     print("hello")


# try:
#     print(10/2)
#     print(10/5)
#     print(20/2)
# except:
#     print("hello")
# print("hi")


# try:
#     print(10/2)
#     print(10/0)
#     print(20/2)
# except ZeroDivisionError:
#     print("hello")
# print("hi")


# try:
#     print(10/2)
#     print(10/0)
#     print(20/2)
# except ValueError:
#     print("hello")
# print("hi")


# try:
#     print(10/2)
#     print(10/0)
#     print(20/2)
# except ZeroDivisionError:
#     print(10/0)
# print("hi")



# try:
#     if x%2==0:
#         print("ok")
# except NameError:
#     print("hello")
# print("hi")

try:
    a=int(input('enter '))
    if x%2==0:
        print("ok")
except ValueError:
    print("hello")
except NameError:
    print("hi")


# try:
#     a=int(input('enter '))
#     if x%2==0:
#         print("ok")
# except (NameError,Exception):
#     print("hello")







